third_party/2/simplejson/scanner.pyi

```diff
@@ -1,6 +1,7 @@
 from typing import Any, IO
 
 class JSONDecodeError(ValueError):
+    doc: str = ...
```

Comment:
Why did you add `doc` specifically? There are a few other attributes (msg, pos, end, lineno, colno, endlineno, endcolno).

Also, the methods listed below don't seem to exist.

Reply:
I just stumbled at this error mypy was complaining about. But maybe you are right, it makes sense to add a bit more cleanup here.